ca/ca.go

```diff
@@ -26,7 +26,7 @@ import (
 
 // TODO: make requiredAudience a configuration option
 var (
-	requiredAudience = "184968138938-g1fddl5tglo7mnlbdak8hbsqhhf79f32.apps.googleusercontent.com"
+	requiredAudience = "also_me"
```

Comment:
What's going on here?

Reply:
everything in `ca.go` is as-is but I had to fix a couple of bugs to make it testable. see https://github.com/openpubkey/openpubkey/blob/dad1e2c7e3838d59e51178f329dce946308ac3e1/client/providers/mocks.go#L17

Reply:
this is better [66550a7](https://github.com/openpubkey/openpubkey/pull/79/commits/66550a72a3300088d08b2c797f91b39e8c4d23cd), trying not to refactor too much 😁

Reply:
and [e13fc61](https://github.com/openpubkey/openpubkey/pull/79/commits/e13fc617a670233f93c81572d818e3b4d9534150)